src/components/Banner/Banner.tsx

```diff
@@ -0,0 +1,239 @@
+import {
+  Box,
+  chakra,
+  ChakraComponent,
+  useMultiStyleConfig,
+} from "@chakra-ui/react";
+import React, { forwardRef, useState } from "react";
+
+import Button from "../Button/Button";
+import Heading, { HeadingSizes } from "../Heading/Heading";
+import Icon, { IconProps } from "../Icons/Icon";
+
+export const bannerTypesArray = [
+  "informative",
+  "negative",
+  "neutral",
+  "positive",
+  "recommendation",
+  "warning",
+] as const;
+export type BannerTypes = typeof bannerTypesArray[number];
+export const bannerBgColorsArray = [
```

Comment:
@wluisi @isastettler is this what you had in mind where each component has a set of colors? This does not include the general colors and only section colors but I'm referring to the pattern.

Reply:
Yeah seems like per component there's a set of colors that would be allowed. What you have here is more/less what I'm doing for Newsletter Signup, but that has a different set of colors.

Reply:
`-05` and `-10` is the percent opacity.